cwl/src/main/scala/cwl/preprocessor/CwlPreProcessor.scala

```diff
@@ -345,6 +376,37 @@ class CwlPreProcessor(saladFunction: BFile => Parse[String] = saladCwlFile) {
     }
   }
 
+  /**
+    * Given a json, collects all "steps.run" values that are JSON Objects representing a workflow.
+    * A saladed JSON is assumed.
+    * @return a Map[String, Json], where the key is the cwl id of the workflow, and the value its content
+    */
+  private def findRunInlinedWorkflows(json: Json): ErrorOr[Map[String, Json]] = {
+    import cats.instances.list._
+    import cats.syntax.traverse._
+
+    json.asArray match {
+      case Some(cwls) => cwls.toList
+        // Those shenanigans are because I can't get traverse to work on map directly, so there's some conversion
+        // back and forth between list and map
```

Comment:
🙈

Reply:
They moved it into alleycats because in some universe it violates some law that we don't care about:

https://github.com/typelevel/cats/blob/master/alleycats-core/src/main/scala/alleycats/std/map.scala

I vote we use that lib and then we can de-shenanigan